guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc

```diff
@@ -78,6 +78,8 @@ network:
   config: disabled
 EOM
 ----
++
+Back up the cfg file before you overwrite it.
```

Comment:
I recommend moving this step to "Prerequisites".

Example:

Back up the following files:
* `/etc/cloud/cloud.cfg.d/01_network.cfg`
* etc